packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -73,7 +73,7 @@ function useAddAppMutation(_type: App["type"] | null, allOptions?: UseAddAppMuta
 
     if (!isOmniInstall) {
       gotoUrl(json.url, json.newTab);
-      return;
+      return { setupPending: externalUrl || json.url.endsWith("/setup") };
```

Comment:
@joeauyeung Can you let me know what exact bug it fixes, I would test that flow.

Reply:
When going to the setup pages for certain apps, ex. Apple Cal a toast would appear saying the app was successfully installed even though it wasn't yet. Same thing with apps that went to Oauth pages